### PR TITLE
make an appointment

### PR DESCRIPTION
**Pull request by Esnoan**
tuve que poner la id generada porque la que me da la api no estaba funcionando

### CODE REVIEW

File: src/page/confirmation-page.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { $, ElementFinder } from 'protractor';
+
+export class ConfirmationPage {
+  private confirmationText: ElementFinder;
+
+  constructor() {
+    this.confirmationText = $('div#page-wrapper p');
+
```
Comment (holgiosalos) [inline src/page/confirmation-page.ts]: unneded blank line

File: test/make-appointment.spec.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const addDoctorPage: AddDoctorPage = new AddDoctorPage();
+  const confirmationPage: ConfirmationPage = new ConfirmationPage();
+  const addPatientPage: AddPatientPage = new AddPatientPage();
+  const makeAppointmentPage: MakeAppointmentPage = new MakeAppointmentPage();
+
+  const doctorId = generateUUID();
+  const patientId = generateUUID();
+
+  it('then should be add a doctor', async () => {
+    await browser.get('http://automatizacion.herokuapp.com/aurbano/');
```
Comment (holgiosalos) [inline test/make-appointment.spec.ts]: make the URL constant

File: test/make-appointment.spec.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import {
+  MenuContentPage,
+  AddDoctorPage,
+  ConfirmationPage,
+  AddPatientPage,
+  MakeAppointmentPage
+} from '../src/page';
+import { generateUUID } from '../src/utils/UUID';
+
+describe('Make an appointment', () => {
```
Comment (holgiosalos) [inline test/make-appointment.spec.ts]: It is better if you follow gherking. For example:
Given I want to schedule a meddical appointment (describe on line 11)
and doctor exists in system (it on line 21)
and I create my patient user (it on line 29)
when I schedule the appointment on <Date> 
then I should get a confirmation message 

for the last two steps you should split the **it** function on line 37

**Review by holgiosalos — CHANGES_REQUESTED** [review]
This is a really good approach. However, I would have liked to see API consumption, for example, to add patient or doctors. (This is not required and is not needed for you add any additional change to the ones on this review)

**Review by yesid-lopez — APPROVED** [review]
Great work!

**Review by holgiosalos — APPROVED** [review]
Very interesting approach and great work!

**Pull request merged**